Sources/Controllers/DashboardOnMap/MapSettings/OAMapSettingsTerrainParametersViewController.mm

```diff
@@ -855,10 +978,11 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
         OASimpleTableViewCell *cell = [tableView dequeueReusableCellWithIdentifier:[OASimpleTableViewCell getCellIdentifier]];
         [cell leftIconVisibility:NO];
         [cell descriptionVisibility:NO];
-        cell.selectionStyle = UITableViewCellSelectionStyleNone;
-        cell.titleLabel.text = [_currentPaletteColorItem toHumanString];
-        cell.titleLabel.textColor = UIColorFromRGB(color_extra_text_gray);
-        cell.titleLabel.font = [UIFont preferredFontForTextStyle:UIFontTextStyleFootnote];
+        BOOL isPaletteName = [item.key isEqualToString:@"paletteName"];
+        cell.selectionStyle = isPaletteName ? UITableViewCellSelectionStyleNone : UITableViewCellSelectionStyleDefault;
+        cell.titleLabel.text = isPaletteName ? [_currentPaletteColorItem toHumanString] : item.title;
+        cell.titleLabel.textColor = [item objForKey:@"tintTitle"] ?: UIColorFromRGB(color_extra_text_gray);
```

Comment:
UIColorFromRGB(color_extra_text_gray) - do we have a similar color from the asset?